test/forms/inputs/textarea.test.js

```diff
@@ -98,6 +99,6 @@ test('Textarea passes down forwardedRef to input correctly', () => {
     forwardedRef: inputRef,
   }
 
-  const wrapper = mount(<Textarea {...props} />)
-  expect(wrapper.find('textarea').prop('id')).toEqual(inputRef.current.id)
+  render(<Textarea {...props} />)
+  expect(screen.getByLabelText('Test').id).toEqual(inputRef.current.id)
```

Comment:
`toHaveAttribute('id', ...)`

Reply:
I have literally no memory of writing these 😆 . Ooops!